calico/reference/installation/_api.html

```diff
@@ -4088,4 +4217,10 @@ <h3 id="operator.tigera.io/v1.TyphaDeploymentSpec">TyphaDeploymentSpec
 </tr>
 </tbody>
 </table>
+<h3 id="operator.tigera.io/v1.WAFStatusType">WAFStatusType
```

Comment:
I think we want to remove the documentation of the WAF types in these docs. They are enterprise-only.

I didn't find a way to update our API reference generation to filter out these fields so I just removed them manually